app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +23,24 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".activities.DictionaryActivity"
+            android:label="@string/title_activity_dictionary"
+            android:parentActivityName=".activities.AddNewWordActivity"
+            android:theme="@style/MaterialTheme">
```

Comment:
we don't need to define the theme for each activity, it will automatically inherit the theme defined in the application object